### PR TITLE
docs: fix build

### LINKED ISSUE #197: Latest doc builds are broken
# What

Current master and develop branch are not being published to the docs pages. The core content is still being pulled from an outdated README published through travis.ci. The doc content has broken links and is not properly updated

### PR DESCRIPTION
**Pull request by vsmalladi**
update build docs so that [main ga4gh page](https://ga4gh.github.io/task-execution-schemas/docs/) is up to date. Closes #197

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
           file: openapi/task_execution_service.openapi.yaml
-
   diff:
     name: Show OpenAPI differences relative to target branch
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'pull_request' }}
     steps:
       - name: Check out head branch
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (uniqueg) [inline .github/workflows/ci.yml:23]: Actually, checkout is at v4 now

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
     runs-on: ubuntu-latest
-    # env:
-    #   TRAVIS_BRANCH: ${{ github.event.number }} 
-    #   a trick that builds docs for PRs (with PR number). Does not work for PRs from forks.
     steps:
       - name: Setup Node.js environment
         uses: actions/setup-node@v2.5.1
-        with: 
+        with:
           node-version: 14.x
```
Comment (uniqueg) [inline .github/workflows/build.yml:12]: This looks very old

**Review by uniqueg — COMMENTED** [review]
LGTM. Perhaps might wanna upgrade actions to latest version while at it

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vsmalladi) [reply .github/workflows/ci.yml]: Updated

File: .github/workflows/build.yml (hunk 2 above)
Comment (vsmalladi) [reply .github/workflows/build.yml]: Updated

**Review by uniqueg — APPROVED** [review]

**Pull request merged**